crates/wit-bindgen-rust/src/interface.rs

```diff
@@ -292,9 +284,6 @@ impl <Ctx, T, Tx> Server<Ctx, Tx> for T
             kind, params, name, ..
         } in &funcs_to_export
         {
-            if !matches!(kind, FunctionKind::Freestanding) {
```

Comment:
This should serve all functions except for methods - constructors, statics and freestanding functions are exposed as `<interface>.<resource?>.<name>`, but methods are exposed as `<resource-id>.<name>`

E.g. `myinterface.myresource.mystaticfunc` and `017F22E2-79B0-7CC3-98C4-DC0C0C07398F.mymethod`